sgt-back-end/index.js

```diff
@@ -0,0 +1,133 @@
+const pg = require('pg');
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      res.status(200).json(result.rows);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({ error: 'An unexpected error occured' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const input = req.body;
+  const name = input.name;
+  const course = input.course;
+  const score = input.score;
+  if (score < 0 || score > 100 || Number.isNaN(score)) {
+    res.status(400).json({ error: 'Score must be a positive integer between 0 and 100' });
+    return;
+  }
+  if (!name || !course || !score) {
+    res.status(400).json({ error: 'Must include Name, Course, and Score.' });
+    return;
+  }
```

Comment:
A non-integer `score` in the `0` to `100` range should still result in a `400 Bad Request` (ex: `43.297`).